ister_test.py

```diff
@@ -4136,7 +4109,6 @@ def mock_os_unlink(path):
     finally:
         tempfile.mkstemp = mkstemp_orig
         shutil.copyfileobj = cfo_orig
-        os.unlink = unlink_orig
```

Comment:
Ugh this just makes me note that this was very untested as the COMMAND_RESULTS isn't being verified X(. Not your problem though.